cli/commands/start.go

```diff
@@ -43,13 +43,15 @@ func init() {
 	// stateboard flags
 	addStateboardRunningFlags(startCmd)
 
+	// Disable instance(s) prefix flag
+	addDisablePrefixFlag(startCmd)
```

Comment:
I think we can add this flag without wrapper function - I've used them for adding multiple flags that are the same for several commend.